core/src/main/java/io/confluent/kafka/schemaregistry/storage/KafkaSchemaRegistry.java

```diff
@@ -859,8 +860,10 @@ private void checkRegisterMode(
     }
 
     if (schema.getId() >= 0) {
-      if (getModeInScope(subject) != Mode.IMPORT) {
-        throw new OperationNotPermittedException("Subject " + subject + " is not in import mode");
+      if (getModeInScope(subject) != Mode.IMPORT
```

Comment:
Let's create a methods similar to isReadOnlyMode() to check IMPORT/FORWARD mode.